packages/forms-web-app/src/controllers/examination/file-upload/handleMultipleFileUploadsWithErrors.js

```diff
@@ -0,0 +1,29 @@
+const { multipleFileValidations } = require('./fileValidation');
+const { saveFileToDisk } = require('./fileManagement');
+const { addFileToSession } = require('./fileSessionManagement');
+
+const handleMultipleFileUploadsWithErrors = async (session, fileToUpload) => {
```

Comment:
any improvements would be appreciated. 


We could split the array into good and bad and loop each but that seems too much ?